ipatests/test_cmdline/test_help.py

```diff
@@ -174,3 +174,18 @@ def test_multiline_description():
         api.Backend.cli.run(['trust-add', '-h'])
 
     assert unicode(api.Command.trust_add.doc).strip() in help_ctx.stdout
+
+
+def test_bz1428690():
```

Comment:
Did you keep this commit on purpose or by mistake?
@abbra explained in this [comment]{https://github.com/freeipa/freeipa/pull/3317#discussion_r298028773) from another PR, this call won't work because there is no `ipa backup` command provided by ipa CLI (the command is `ipa-backup`).